### PR TITLE
chore: Minor fixes from #196

### PR DESCRIPTION
**Pull request by lewis-sqa**
# Description

- Fixed `yarn.lock` changes.
- Fixed Angular script.
- Fixed commands in contributing file.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

# Type of change. This type of change is the main reason for the PR.

- [ ] FIX - a PR of this type patches a bug.
- [ ] FEATURE - a PR of this type introduces a new feature.
- [ ] BUILD - a PR of this type introduces build changes.
- [ ] CI - a PR of this type introduces CI changes.
- [ ] DOCS - a PR of this type introduces DOCS improvement.
- [ ] STYLE - a PR of this type introduces style changes.
- [ ] REFACTOR - a PR of this type introduces refactoring.
- [ ] PERFORMANCE - a PR of this type introduces performance changes.
- [ ] TEST - a PR of this type adds more tests.
- [x] CHORE - a PR introduces other changes than the specified above.

# Breaking changes

- [ ] BREAKING CHANGE - SPECIFY: _______
- [x] NO BREAKING CHANGE - this PR doesn't contain any breaking changes and it's backwards compatible

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,16 +10,16 @@ The core modules are in the `packages` directory and the example apps are in the
...
-```
-$ nx run build:near-wallet-selector #build the library
+```bash
+yarn nx run near-wallet-selector:build #build the library
 
-$ nx serve react # to run the react example
-$ nx serve angular # to run the angular example
+yarn nx serve react # to run the react example
+yarn nx serve angular # to run the angular example
 ```
```
Comment (rgolea) [inline CONTRIBUTING.md:25]: Isn't it running the commands properly without the "yarn" prefix?

**Review by rgolea — APPROVED** [review]
I don't really think the `yarn` prefix is necessary since the cli is a dependency. But I don't run windows either so there might be stuff going on there.

File: CONTRIBUTING.md (hunk 1 above)
Comment (lewis-sqa) [reply CONTRIBUTING.md:25]: I suspect you might have `nx` installed globally so it's registered in your PATH? Yarn is quite nice for things like this as it lets you run commands in the local `.bin` directory for instances like this

Comment (lewis-sqa) [reply CONTRIBUTING.md:25]: What does `which nx` give you?

Comment (lewis-sqa) [reply CONTRIBUTING.md:25]: In fact, my PATH has magically got the project .bin in there now. Does NX do this in a post install or something. That's not normal behaviour 🤔

**Pull request merged**